charts/app-config/templates/external-secrets/link-checker-api/basic-auth.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: link-checker-api-basic-auth 
```

Comment:
Are these basic auth secrets just the `betademo` login? If so, we need to have just one secret for them.

Reply:
I was thinking the same - but was not sure if this was the same in PROD?

Reply:
meaning do they have same shared user or seperate?